web/src/components/HistoryViewer.jsx

```diff
@@ -0,0 +1,117 @@
+import { Fragment, h } from 'preact';
+import { useCallback, useEffect, useRef, useState } from 'preact/hooks';
+import { useApiHost, useEvents } from '../api';
+import { useSearchString } from '../hooks/useSearchString';
+import { Next } from '../icons/Next';
+import { Play } from '../icons/Play';
+import { Previous } from '../icons/Previous';
+import { HistoryHeader } from '../routes/HistoryHeader';
+import { longToDate } from '../utils/dateUtil';
+import Timeline from './Timeline';
+
+export default function HistoryViewer({ camera }) {
+  const apiHost = useApiHost();
+  const videoRef = useRef();
+
+  // const beginningOfDay = new Date().setHour(0, 0, 0) / 1000;
+  const beginningOfDay = new Date(new Date().getTime() - 24 * 60 * 60 * 1000);
```

Comment:
I'd like this to be day of, and potentially if you reach the end, load the next days? Or would a dropdown be better?